vendors/renesas/boards/rx65n-rsk/aws_demos/config_files/ota_config.h

```diff
@@ -0,0 +1,207 @@
+ * request is 128/1 = 128 blocks. Configure this parameter to this maximum
+ * limit or lower based on how many data blocks response is expected for each
+ * data requests.
+ *
+ * <b>Possible values:</b> Any unsigned 32 integer value greater than 0. <br>
+ */
+#define otaconfigMAX_NUM_BLOCKS_REQUEST         4U
+
+/**
+ * @brief The maximum number of requests allowed to send without a response
+ * before we abort.
+ *
+ * @note This configuration parameter sets the maximum number of times the
+ * requests are made over the selected communication channel before aborting
+ * and returning error.
+ *
+ * <b>Possible values:</b> Any unsigned 32 integer. <br>
+ */
+#define otaconfigMAX_NUM_REQUEST_MOMENTUM       32U
+
+/**
+ * @brief How frequently the device will report its OTA progress to the cloud.
+ *
+ * @note Device will update the job status with the number of blocks it has
+ * received every certain number of blocks it receives. For example, 64 means
+ * device will update job status every 64 blocks it receives.
+ *
+ * <b>Possible values:</b> Any unsigned 32 integer. <br>
+ */
+#define otaconfigOTA_UPDATE_STATUS_FREQUENCY    25U
+
+/**
+ * @brief The number of data buffers reserved by the OTA agent.
+ *
+ * @note This configurations parameter sets the maximum number of static data
+ * buffers used by the OTA agent for job and file data blocks received.
+ *
+ * <b>Possible values:</b> Any unsigned 32 integer. <br>
+ */
+#define otaconfigMAX_NUM_OTA_DATA_BUFFERS       4U
+
+/**
+ * @brief Flag to enable booting into updates that have an identical or lower
+ * version than the current version.
+ *
+ * @note Set this configuration parameter to '1' to disable version checks.
+ * This allows updates to an identical or lower version. This is provided for
+ * testing purpose and it's recommended to always update to higher version and
+ * keep this configuration disabled.
+ *
+ * <b>Possible values:</b> Any unsigned 32 integer. <br>
+ */
+#define otaconfigAllowDowngrade                 0U
+
+/**
+ * @brief The protocol selected for OTA control operations.
+ *
+ * @note This configurations parameter sets the default protocol for all the
+ * OTA control operations like requesting OTA job, updating the job status etc.
+ *
+ * @note Only MQTT is supported at this time for control operations.
+ *
+ * <b>Possible values:</b> OTA_CONTROL_OVER_MQTT <br>
+ */
+#define configENABLED_CONTROL_PROTOCOL          ( OTA_CONTROL_OVER_MQTT )
+
+/**
+ * @brief The protocol selected for OTA data operations.
+ *
+ * @note This configurations parameter sets the protocols selected for the data
+ * operations like requesting file blocks from the service.
+ *
+ * <b>Possible values:</b><br>
+ * Enable data over MQTT - ( OTA_DATA_OVER_MQTT ) <br>
+ * Enable data over HTTP - ( OTA_DATA_OVER_HTTP ) <br>
+ * Enable data over both MQTT & HTTP - ( OTA_DATA_OVER_MQTT | OTA_DATA_OVER_HTTP ) <br>
+ */
+#define configENABLED_DATA_PROTOCOLS            ( OTA_DATA_OVER_MQTT )
```

Comment:
Why HTTP is not enabled here?

Reply:
I have updated this to enable it. Thanks.